controllers/s3utils.go

```diff
@@ -400,6 +400,13 @@ func (s *s3ObjectStore) listKeys(bucket string, keyPrefix string) (
 			ContinuationToken: nextContinuationToken,
 		})
 		if err != nil {
+			//nolint:errorlint
+			if aerr, ok := err.(awserr.Error); ok {
```

Comment:
I should probably do [this](https://pkg.go.dev/errors#As) to get the wrapped error in higher layers.

Reply:
An alternate long term approach may be to use the `createBucket` interface in the prepare phase. The `createBucket` method does NOT return an error if the bucket already exists.  With this approach, you wouldn't have to change the listKeys method at all.


If you prefer to stay the course with modifying the listKeys for now, you can take a look at PR #238 that coincidentally now adds a method to check if an error [isAwsErrCodeNoSuchBucket()](https://github.com/RamenDR/ramen/pull/238/commits/c6ed6d4882af10ddc92a49a276a0ae8c894c5947#diff-c4ea3ec52e9a599a126fddfa9e3194b7d329ee6812b23197e810a3747ecc0be5R591). You may want to add a TODO to revisit this later and change the function header to mention that an error is not returned when bucket doesn't exist.

Reply:
Ack, applied the same to this check and moved it up to downloadPVs at least for now.